Library/Homebrew/test/rubocops/text_spec.rb

```diff
@@ -215,5 +215,53 @@ def install
         end
       RUBY
     end
+
+    it "When make calls are not separated" do
+      expect_offense(<<~RUBY)
+        class Foo < Formula
+          def install
+            system "make && make install"
+            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use separate `make` calls
+          end
+        end
+      RUBY
+    end
+
+    it "When a single inreplace is in block form" do
+      expect_offense(<<~RUBY)
+        class Foo < Formula
+          def install
+            inreplace "foo" do |s|
+            ^^^^^^^^^^^^^^^^^^^^^^ 'inreplace ... do' was used for a single substitution (use the non-block form instead).
+              s.gsub! "bar", "baz"
+            end
+          end
+        end
+      RUBY
+    end
+  end
+end
+
+describe RuboCop::Cop::FormulaAuditStrict::Text do
+  subject(:cop) { described_class.new }
+
+  context "When auditing formula text" do
+    it "when deprecated `env :userpaths` is present" do
+      expect_offense(<<~RUBY)
+        class Foo < Formula
+          env :userpaths
+          ^^^^^^^^^^^^^^ `env :userpaths` in homebrew/core formulae is deprecated
+        end
+      RUBY
+    end
+
+    it "when deprecated `env :std` is present in homebrew-core" do
+      expect_offense(<<~RUBY, "/homebrew-core/")
+        class Foo < Formula
+          env :std
+          ^^^^^^^^ `env :std` in homebrew/core formulae is deprecated
+        end
+      RUBY
+    end
```

Comment:
These tests fail and I'm not sure why. In each case, `rubocop` runs as expected when I run it on a file that I've put bad `env` lines in.

The tests, however, fail unless the `env :std` or `env :userpaths` is placed inside `def install ... end`. In looking at previous commits (e.g. https://github.com/Homebrew/homebrew-core/commit/6608e76ebf9e7e5b58341f6333e0717db126678a or https://github.com/Homebrew/homebrew-core/commit/5475dad85e0b58def66a3bd2c8d23943caf8082b), it seems that the `env` lines would not have gone in the `install` block)

Reply:
Fixed!